certified-connectors/Dokobit Portal/apiDefinition.swagger.json

```diff
@@ -1462,6 +1601,401 @@
+              "required": ["file"],
+              "properties": {
+                "file": {
+                  "type": "object",
+                  "required": ["name", "content"],
+                  "properties": {
+                    "name": {
+                      "type": "string",
+                      "description": "File name.",
+                      "x-ms-summary title": "Name",
+                      "x-ms-visibility": "important"
+                    },
+                    "content": {
+                      "type": "string",
+                      "description": "Base64 encoded file content."
+                      }
+                    }
+                  }
+                }
+              },
+            "x-ms-visibility": "important"
+            }
+        ],
+        "responses": {
+          "200": {
+            "description": "200",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "status": {
+                  "type": "string",
+                  "description": "Upload status"
+                },
+                "token": {
+                  "type": "string",
+                  "description": "Validation token",
+                  "title": "token"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "400",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "status": {
+                  "type": "string",
+                  "description": "status"
+                },
+                "message": {
+                  "type": "string",
+                  "description": "message"
+                },
+                "errors": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "message": {
+                        "type": "string",
+                        "description": "message"
+                      },
+                      "error_code": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "error_code"
+                      },
+                      "field": {
+                        "type": "string",
+                        "description": "field"
+                      }
+                    }
+                  },
+                  "description": "errors"
+                },
+                "error_code": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "error_code"
+                }
+              }
+            }
+          }
+        },
+        "x-ms-visibility": "important"
+      }
+    },
+    "/validation/{token}/download/detailed-report-pdf": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {}
```

Comment:
@epelanis empty schema is not allowed. please define a proper schema response. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses).
Please fix all empty schemas in the swagger file.

Reply:
@Amjed-Ayoub Could you suggest a description in cases where we return a file?

Reply:
Oh I just noticed this sorry.
@epelanis you can just define a type for the schema. pelase see below for example.
```
"responses": {
          "200": {
            "description": "default",
            "schema": {
                    "type": "string"
            }
           }
  }
```

Reply:
@Amjed-Ayoub Tried testing with "type": "string", but it did not work in all cases. Found that "type": "file" would be acceptable.

Reply:
@epelanis I am sorry if I was not very clear before. the schema type should match with the response schema type. it could be any type, string, object, array, number...etc.

Reply:
@Amjed-Ayoub Thanks. Sorry for the misunderstanding. I updated the schemas again.